traitsui/tests/test_actions.py

```diff
@@ -149,10 +189,7 @@ def test_qt_button_action(self):
 
     # ----- wx tests
 
-    @unittest.skipIf(
-        not is_mac_os,
-        "Problem with triggering toolbar actions on Linux and Windows. Issue #428.",  # noqa: E501
-    )
+    @unittest.skip("Problem with triggering toolbar actions. Issue #428.")
```

Comment:
why did we remove macOS testing here?

Reply:
Digging into this a little to see what is going on, and looking at the UI generated, there is a toolbar, but no toolbar buttons show.